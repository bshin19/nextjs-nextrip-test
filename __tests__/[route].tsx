import { render } from "@testing-library/react";
import Route, { getServerSideProps } from "../pages/route/[route]";

describe("Route", () => {
  it("should match default", () => {
    const { container } = render(<Route />);
    expect(container).toMatchSnapshot();
  });
});

describe("getServerSideProps", () => {
  const context = { params: { route: 100 } };
  it("should return the appropriate props", async () => {
    expect(await getServerSideProps(context as any)).toMatchSnapshot();
  });
});
