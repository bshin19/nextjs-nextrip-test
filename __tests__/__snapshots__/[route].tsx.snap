// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route should match default 1`] = `<div />`;

exports[`getServerSideProps should return the appropriate props 1`] = `
Object {
  "props": Object {
    "directions": Array [
      Object {
        "direction_id": "0",
        "direction_name": "Northbound",
      },
      Object {
        "direction_id": "1",
        "direction_name": "Southbound",
      },
    ],
  },
}
`;
